net-print/gutenprint/gutenprint-5.3.4.ebuild

```diff
@@ -0,0 +1,84 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+inherit autotools
```

Comment:
Since you're most likely going to have to edit this ebuild, slap an empty newline between these lines here :P